husky_components/tabs/main.js

```diff
@@ -50,6 +50,7 @@ define(function() {
             forceReload: false,
             callback: null,
             forceSelect: true,
+            conditionData: {},
```

Comment:
but that is not the "data" which is used to render the component - at least in all other components we use the term "data" like this.